Frends.PDF.Create/Frends.PDF.Create/Frends.PDF.Create.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <Version>1.0.0</Version>
+    <Authors>Frends</Authors>
+	<Copyright>Frends</Copyright>
+	<Company>Frends</Company>
+	<Product>Frends</Product>
+	<PackageTags>Frends</PackageTags>
+	<PackageLicenseExpression>MIT</PackageLicenseExpression>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	<Description>FRENDS task to create PDF document.</Description>
```

Comment:
Either "a PDF document" or "PDF documents" (minor spelling bug).

Reply:
Changed to documents.